src/index.html

```diff
@@ -1,14 +1,203 @@
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Human rights</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 class="title">Universal Declaration of Human Rights</h1>
+    <h2 class="subtitle">Preamble</h2>
+    <p>
+    Whereas recognition of the inherent dignity and of the equal and inalienable
+    rights of all members of the human family is the foundation of freedom, justice
+    and peace in the world,
+      <br>
+      Whereas disregard and contempt for human rights have resulted in barbarous
+      acts which have outraged the conscience of mankind, and the advent of a world
+      in which human beings shall enjoy freedom of speech and belief and freedom
+      from fear and want has been proclaimed as the highest aspiration of the common
+      people,
+      <br>
+      Whereas it is essential, if man is not to be compelled to have recourse, as a last
+      resort, to rebellion against tyranny and oppression, that human rights should be
+      protected by the rule of law,
+      <br>
+      Whereas it is essential to promote the development of friendly relations between
+      nations,
+      <br>
+      Whereas the peoples of the United Nations have in the Charter reaffirmed their
+      faith in fundamental human rights, in the dignity and worth of the human person
+      and in the equal rights of men and women and have determined to promote
+      social progress and better standards of life in larger freedom,
+      <br>
+      Whereas Member States have pledged themselves to achieve, in cooperation
+      with the United Nations, the promotion of universal respect for and observance of
+      human rights and fundamental freedoms,
+      <br>
+      Whereas a common understanding of these rights and freedoms is of the
+      greatest importance for the full realization of this pledge,
+      <br>
+      Now, therefore,
+      <br>
+      The General Assembly,
+      <br>
+      Proclaims this Universal Declaration of Human Rights as a common standard of
+      achievement for all peoples and all nations, to the end that every individual and
+      every organ of society, keeping this Declaration constantly in mind, shall strive by
+      teaching and education to promote respect for these rights and freedoms and by
+      progressive measures, national and international, to secure their universal and
+      effective recognition and observance, both among the peoples of Member States
+      themselves and among the peoples of territories under their jurisdiction.
+    </p>
+    <h2 class="subtitle">Article 1</h2>
+    <p>
+    All human beings are born free and equal in dignity and rights. They are
+    endowed with reason and conscience and should act towards one another in a
+    spirit of brotherhood.
+    </p>
```

Comment:
Same here and through the document.
```suggestion
    <p>
      All human beings are born free and equal in dignity and rights. They are
      endowed with reason and conscience and should act towards one another in a
      spirit of brotherhood.
    </p>
```